prac_06/languages.py

```diff
@@ -0,0 +1,20 @@
+# Languages
+"""
+Languages
+Estimate: 15 minutes
+Actual:   10 minutes
+"""
+from prac_06.programming_language import ProgrammingLanguage
+
+def main():
+    python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+    ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+    visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+
+    languages = [python, ruby, visual_basic]
+    print("The dynamically typed languages are:")
+
+    dynamic_language = [language for language in languages if language.is_dynamic()]
```

Comment:
yes this works but there is a slightly more efficient way to use is .is_dynamic  in this instance (It works though so that's more important!)